e2e/src/test/groovy/gov/hhs/cdc/trustedintermediary/e2e/MetadataTest.groovy

```diff
@@ -17,7 +18,6 @@ class MetadataTest extends Specification {
         given:
         def expectedStatusCode = 200
         def inboundSubmissionId = UUID.randomUUID().toString()
-        def outboundSubmissionId = "1234567890"
```

Comment:
Given you only have one submission id in the test that should work for inbound and outbound regardless, I think it's better to rename `inboundSubmissionId` to `submissionId`. Same for the other tests

Reply:
The test will change to include the `outboundSubmissionId` based on the comment below.